webapp/src/shared/components/Text/Text.tsx

```diff
@@ -28,21 +29,29 @@ export enum TextWeight {
 type TextProps = {
   variant: TextVariant;
   color?: TextColor;
+  parent_class?: string;
```

Comment:
agree with this

Reply:
This one will wait for #96.

Reply:
Also, I think we should avoid passing classes to components, we should respect as much as we can the interfaces exposed by the components (aka props) because they define how the component, from how we conceived it, can be modified. If we pass classes we kind of bypassing those restrictions and let te component be modified how we want.

In this case I'd say you could simply just create a `div`wrapper around the text component and apply all styling you need to it.

Reply:
We have a mexican standoff here:

gbudau:
> praise: I like how you use the avatar props for this component.
I obviously don't dislike the idea.

Reply:
There are pros and cons to both options :smiley: 

If we don't add a `className` prop to the components, we have to create wrappers to apply custom styling and positioning. It might make the code a bit less legible.

If we pass a `className` prop, we don't have to create wrappers, it might make the code more legible, but we can shoot ourselves in the foot.

Reply:
Let me explain this from another point of view:

If we enable classes to be passed around we'll be opening the component to be modified anywhere just to solve a particular case. For me, having a `div` around a specific component in a few places does not worsen legibility :stuck_out_tongue_closed_eyes: 

Another reason for not passing around classes is we would have css for the same component defined in several places (some styling defined near to it, other styling near to the component which needs it to behave different) You may have problems identifying sometimes where that CSS comes from.

In component libraries, such as Material UI, what they expose is [a utility](https://mui.com/system/styled/) to apply styles to components in order to create new ones, variations. But when you have a component, you want to respect as much as you can exposed interfaces. That component has been conceived to behave in a specific way.

If you need to start tweaking or hacking it, it may be a sign of a need to create a new variantion component.